src/Database/MorphPivot.php

```diff
@@ -148,7 +148,7 @@ public function newQueryForRestoration($ids)
         }
 
         if (!str_contains($ids, ':')) {
-            return parent::newQueryForRestoration($ids);
+            return parent::newQueryForRestoration([$ids]);
```

Comment:
This matches [Laravel's code](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Eloquent/Relations/MorphPivot.php#L130-L152) exactly, so I'm fine not making this change. Really what should be changed is the docblock in Laravel's code.